src/app/tests/suites/TestGroupKeyManagementCluster.yaml

```diff
@@ -179,3 +123,28 @@ tests:
       attribute: "maxGroupKeysPerFabric"
       response:
           value: 1
+
+    - label: "KeySet Remove"
+      command: "KeySetRemove"
+      arguments:
+          values:
+              - name: "groupKeySetID"
+                value: 101
+
+    - label: "KeySet Read (removed)"
+      command: "KeySetRead"
+      arguments:
+          values:
+              - name: "groupKeySetID"
+                value: 101
+      response:
+          error: 0x8B
+
+    - label: "KeySet Read (removed)"
+      command: "KeySetRead"
+      arguments:
+          values:
+              - name: "groupKeySetID"
+                value: 101
```

Comment:
What is the point of doing this same read twice here?

Reply:
Not sure why this test got duplicated. Removed.